style/javascript/README.md

```diff
@@ -0,0 +1,18 @@
+JavaScript
+==========
+
+[Sample](sample.js)
+
+* Prefer ES6 classes over library provided classes.
+* Prefer `===` and `!===` to `==` and `!=`.
+* Avoid using arrow functions when not necessary.
```

Comment:
I see the value in being explicit about using `this`, but I don't like giving up on the clarity arrow functions can provide with implicit return and shorter syntax

``` es6
users.map(user => user.name)

// vs
users.map(function(user) { return user.name })
```